_locales/de/messages.json

```diff
@@ -716,15 +716,15 @@
 		"message": "E-Mail muss von der spezifizierten SDID signiert sein."
 	},
 	"ruletype.2.description": {
-		"message": "E-Mail muss nicht signiert sein. Falls Sie signiert ist, muss Sie von der spezifizierten SDID signiert sein."
+		"message": "E-Mail muss nicht signiert sein. Wenn Sie signiert ist, muss sie von der spezifizierten SDID signiert sein."
```

Comment:
Is there a reason to make the second `sie` lowercase, but keep the first one uppercase? Same question below for `ruletype.3.description`.

Reply:
Sorry, I overlooked it.